config/samples/core_v1beta1_featureflagsource.yaml

```diff
@@ -1,13 +1,22 @@
 apiVersion: core.openfeature.dev/v1beta1
+kind: FeatureFlag
```

Comment:
why it's needed to add FF sample in FFS sample?

Reply:
that's a leftover from trying something out earlier - will remove it again